packages/session-replay-browser/src/config/remote-config.ts

```diff
@@ -13,6 +13,7 @@ import {
 const UNEXPECTED_NETWORK_ERROR_MESSAGE = 'Network error occurred, session replay remote config fetch failed';
 const SUCCESS_REMOTE_CONFIG = 'Session replay remote config successfully fetched';
 const MAX_RETRIES_EXCEEDED_MESSAGE = 'Session replay remote config fetch rejected due to exceeded retry count';
+const TIMEOUT_MESSAGE = 'Session replay remote config fetch rejected due to timeout';
```

Comment:
Nit: It'd be nice to include the timeout duration. Something like "... rejected due to timeout (after X seconds)", just so it's really obvious on the timeout duration.